org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/MapExtensions.java

```diff
@@ -413,6 +413,8 @@ public boolean apply(K input) {
 	 * @return a map with equal keys but transformed values. Never <code>null</code>.
 	 * @since 2.4
 	 */
+	@Pure
+	@Inline(value = "($3.transformValues($1, new $4<$7, $8>($2)))", imported = {Maps.class, FunctionDelegate.class}, constantExpression = true)
```

Comment:
What is the added value of the `@Inline` here?

Reply:
Give the opportunity to Java compiler to apply optimizations.
At the end, the inlining is not mandatory; and could be removed safely.